pkg/filesystem/filesystem.go

```
@@ -146,37 +148,47 @@ func mountRootfs(ipList []string, target string, cluster *v2.Cluster, initFlag b
 	}*/
-	var wg sync.WaitGroup
+	g := new(errgroup.Group)
 	config := runtime.GetRegistryConfig(
 		common.DefaultTheClusterRootfsDir(cluster.Name),
 		runtime.GetMaster0Ip(cluster))
 	src := common.DefaultMountCloudImageDir(cluster.Name)
+	renderEtc := filepath.Join(common.DefaultMountCloudImageDir(cluster.Name), common.EtcDir)
+	renderChart := filepath.Join(common.DefaultMountCloudImageDir(cluster.Name), common.RenderChartsDir)
+	renderManifests := filepath.Join(common.DefaultMountCloudImageDir(cluster.Name), common.RenderManifestsDir)
 	// TODO scp sdk has change file mod bug
 	initCmd := fmt.Sprintf(RemoteChmod, target)
 	envProcessor := env.NewEnvProcessor(cluster)
 	for _, IP := range ipList {
-		wg.Add(1)
-		go func(ip string) {
-			defer wg.Done()
-			sshClient, err := ssh.GetHostSSHClient(ip, cluster)
+		g.Go(func() error {
+			for _, dir := range []string{renderEtc, renderChart, renderManifests} {
+				if utils.IsExist(dir) {
+					err := envProcessor.RenderAll(IP, dir)
+					if err != nil {
+						return err
+					}
+				}
+			}
+			sshClient, err := ssh.GetHostSSHClient(IP, cluster)
 			if err != nil {
-				errCh <- fmt.Errorf("get host ssh client failed %v", err)
-				return
+				return fmt.Errorf("get host ssh client failed %v", err)
 			}
-			err = CopyFiles(sshClient, ip == config.IP, ip, src, target)
+			err = CopyFiles(sshClient, IP == config.IP, IP, src, target)
 			if err != nil {
-				errCh <- fmt.Errorf("copy rootfs failed %v", err)
-				return
+				return fmt.Errorf("copy rootfs failed %v", err)
 			}
 			if initFlag {
-				err = sshClient.CmdAsync(ip, envProcessor.WrapperShell(ip, initCmd))
+				err = sshClient.CmdAsync(IP, envProcessor.WrapperShell(IP, initCmd))
 				if err != nil {
-					errCh <- fmt.Errorf("exec init.sh failed %v", err)
+					return fmt.Errorf("exec init.sh failed %v", err)
 				}
 			}
-		}(IP)
+			return err
+		})
+	}
+	if err := g.Wait(); err == nil {
+		fmt.Println(err)
```

You should return the error